readme.md

```diff
@@ -330,52 +311,59 @@ folders if they do not exist.
 
 --------------------------------------------------------------------------------
 
-## If time allows - aliases in your bash profile
+## Bash Profile Aliases (If Time Allows)
+
+You may have noticed during Installfest that the instructors messed with this file: `~/.bash_profile`.
 
-You may have noticed during installfest that the instructors messed with the `~/.bash_profile` file. During installfest, we changed the  `~/.bash_profile` to change your prompt to a better one! Turns out, there's a lot you can do with the  `~/.bash_profile`.
+> Based on the path, where is this file located?
 
-There will be commands that you'll find yourself doing frequently it may become a pain to continue to type commands out all the time. It would be really nice if we could shorten some of these commands... enter aliasing. Aliasing is really quite simple!
+Essentially, we changed the  `~/.bash_profile` to make your prompt into a better one!
 
-Let's open our  `~/.bash_profile` in atom and type in the following:
+There will be commands you will find yourself doing frequently. It might become a pain to type out these commands in full all the time. It would be really nice if we could shorten some of these commands... enter aliasing. Aliasing is really quite simple!
+
+<!-- AM: Opportunity for industry experience - show aliases that I and other instructors use. -->
+<!-- AM: Another one -- sometimes we alias things that aren't that long (e.g., ls -al to ls). As developers, we're always trying to find the shortest way to do something. -->
+
+Let's open our  `~/.bash_profile` in atom and type in the following...
 
 ```
 alias greeting="echo 'hello world'"
 alias gs='git status'
 ```
 
-### You do - Make an alias
-Take the next 5 minutes to create your own alias and test it!
+### You Do: Make An Alias
 
-Try and alias something you think you'll find yourself doing frequently!
+Take the next five minutes to create your own alias and test it. Alias something you think you'll find yourself doing frequently!
 
-## Homework
-### To OZ
+--------------------------------------------------------------------------------
+
+## Homework: To Oz
```

Comment:
Going off of what was taught in wdi9 (and I believe wdi8). Is there another hw you have in mind @jshawl?